.eslintrc.js

```diff
@@ -9,7 +12,7 @@ module.exports = {
 	},
 	"parserOptions": { "ecmaVersion": 2017 },
 	"rules": {
-		"quotes": ["error", "double"],
+		"prettier/prettier": "warn",
```

Comment:
More rules can go away because, Prettier is taking care of them

```

no-extra-semi
semi
brace-style
eol-last
no-empty
no-multiple-empty-lines
no-multi-spaces
no-trailing-spaces
key-spacing
object-curly-spacing
indent
```

Reply:
Better use https://github.com/prettier/eslint-config-prettier#cli-helper-tool, prettier sometimes add something new that can cross with eslint rules

Reply:
and that...